spec/fixtures/apps/crash/node-extra-args.js

```diff
@@ -0,0 +1,10 @@
+const path = require('path');
+const childProcess = require('child_process');
+const argv = process.argv;
+let exitCode = 0;
+if (argv.length !== 3) {
```

Comment:
this test passes no matter what :/ i changed this number to 13 and still got a pass.

Reply:
Oh yuck; will take a look

Reply:
Actually, took a look at the existing Node tests - I think we may already be checking for preserving process.argv on child_process.fork 👀 https://github.com/electron/electron/blob/main/spec/node-spec.ts#L41

I'll rip out my bad test and take a look at this one

Reply:
gotta make sure we're checking that w/ the crash reporter enabled, though, since now we're only passing that command-line flag through in that situation.

Reply:
Fixed in [9d3bedb](https://github.com/electron/electron/pull/36460/commits/9d3bedb469b57443ed27380d83a55f4efa1d8edf) - tested the failed path with args that didn't match, and it failed as expected 🙇‍♀️